### PR TITLE
CI/DOC: warning in docbuild

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

**Comment by jreback** [comment]
merge in green

**Comment by jbrockmendel** [comment]
```
----> 1 pd.read_csv(io.StringIO(data), true_values=["Yes"], false_values=["No"])

TypeError: initial_value must be str or None, not DataFrame
```

Maybe we need to freeze the old example so it doesnt get evaluated?  i dont know how that works

**Comment by jreback** [comment]
use a ioython block and paste the example in

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
Thx for fixing

**Comment by jbrockmendel** [comment]
@jreback green but im really not confident when it comes to code-block stuff

File: doc/source/whatsnew/v0.10.0.rst [hunk 1]
```diff
@@ -181,6 +181,7 @@ labeled the aggregated group with the end of the interval: the next day).
   ``X0``, ``X1``, ...) can be reproduced by specifying ``prefix='X'``:
 
 .. ipython:: python
+   :okwarning:
```
Comment (jreback) [inline doc/source/whatsnew/v0.10.0.rst:184]: prob dont' need this now, but nbd.

**Comment by jreback** [comment]
don't need the okwarning but ok

**Pull request merged**